### DISCUSSION TITLE
Unable to install through Docker Container

### DISCUSSION THREAD

**Discussion by kssathish6** [alexjustesen/speedtest-tracker#527] [Q&A]
[/usr/share/doc/adduser/examples/adduser.local.conf](docker: failed to register layer: unlinkat /usr/share/doc/adduser/examples/adduser.local.conf.examples: invalid argument.)

I am getting the above error message. I am new to docker. Trying this on proxmox lxc Ubuntu OS

**Comment by MAMedici** [comment]
I'm getting the same error here.

`
root@docker:~# docker run -d --name speedtest-tracker --restart unless-stopped -p 8280:80 -p 8243:443 -e PUID=1000 -e PGID=1000 -v /var/lib/docker/volumes/stt-config/:/config -v /var/lib/docker/volumes/stt-web/:/etc/ssl/web ghcr.io/alexjustesen/speedtest-tracker:latest

Unable to find image 'ghcr.io/alexjustesen/speedtest-tracker:latest' locally

latest: Pulling from alexjustesen/speedtest-tracker
2ab09b027e7f: Pull complete
9a37c607e3b8: Pull complete
d668f263769c: Pull complete
a5c6dc5d8e21: Pull complete
d5456ce17fb3: Pull complete
ea52adb0571d: Extracting  37.33MB/37.33MB
ee44318289b9: Download complete
5e97411880a4: Download complete
057b58475198: Download complete
cad9848302a0: Download complete
e30c1f8cfae7: Download complete
2fd16ee8822a: Download complete
0d3f314d0464: Download complete
395bdd69c040: Download complete
d468eb6d8919: Download complete
203dcbc0a220: Download complete
0d1dbb22b53a: Download complete
b3c70c6cab3e: Download complete
4f4fb700ef54: Download complete
7c390bcc7320: Download complete
637f230e074e: Download complete
docker: failed to register layer: ApplyLayer exit status 1 stdout:  stderr: unlinkat /usr/share/doc/adduser/examples
adduser.local.conf.examples: invalid argument.
See 'docker run --help'.
`

**Comment by alexjustesen** [comment]
Try pulling the image from Docker hub in case there was some issue with GHCR
